lms/static/js/jquery.gradebook.js

```diff
@@ -27,7 +27,8 @@ var Gradebook = function($element) {
 
 	var moveDrag = function(e) {
 		var offset = e.pageX - mouseOrigin;
-		var targetLeft = clamp(tableOrigin + offset, -maxScroll, 0);
+		// Here updating direction of scroll.
```

Comment:
yes I think it is explaining what code is currently doing. My comment here do not cover past code.

I am saying here the this section of code is updating direction of scroll here